From 7ad2ee1faef99d23b8e36115b0790728276d839e Mon Sep 17 00:00:00 2001
From: David Heidelberg <david@ixit.cz>
Date: Fri, 12 Dec 2025 16:34:33 +0100
Subject: [PATCH 15144/15333] arm64: dts: qcom: sdm845-google: Enable fuel
 gauge

Enable the fuel gauge and configure the associated charger and battery.

Signed-off-by: David Heidelberg <david@ixit.cz>
---
 arch/arm64/boot/dts/qcom/sdm845-google-common.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-google-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-google-common.dtsi
index c2b894d59707..8df2d0b485bb 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-google-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-google-common.dtsi
@@ -454,6 +454,13 @@ &pmi8998_charger {
 	status = "okay";
 };
 
+&pmi8998_fg {
+	monitored-battery = <&battery>;
+	power-supplies = <&pmi8998_charger>;
+
+	status = "okay";
+};
+
 &pmi8998_haptics {
 	qcom,wave-play-rate-us = <6667>;
 
-- 
2.55.0

